vmm/src/config.rs

```diff
@@ -143,7 +146,7 @@ impl OptionParser {
     }
 
     pub fn convert<T: FromStr>(&self, option: &str) -> OptionParserResult<Option<T>> {
-        match self.options.get(option).and_then(|v| v.value.as_ref()) {
+        match self.get(option) {
```

Comment:
Nit: This patch could be squashed with the first one (when introducing `OptionParser` along with its implementation).

Reply:
I intentionally left that as a separate commit to show the "discovery" of the unusual behaviour.

Reply:
Ok!